{"name": "test_valid_auth[performance_glitch_user-secret_sauce]", "status": "broken", "statusDetails": {"message": "TypeError: 'str' object is not callable", "trace": "self = <tests.test_auth.TestAuth object at 0x0000012C0684E490>, email = 'performance_glitch_user', password = 'secret_sauce'\napp = <pages.application.Application object at 0x0000012C06852B80>\n\n    @allure.story(\"Авторизация\")\n    @allure.severity(\"blocker\")\n    @pytest.mark.parametrize(\n        \"email, password\",\n        (\n                (Users.STANDARD_USERNAME, Users.PASSWORD),\n                (Users.PROBLEM_USERNAME, Users.PASSWORD),\n                (Users.PER_GLITCH_USERNAME, Users.PASSWORD)\n        ),\n    )\n    def test_valid_auth(self, email, password, app):\n        \"\"\"\n        1. Открыть страницу\n        2. Ввести валидные значения в поля логина и пароля\n        3. Нажать кнопку LOGIN\n        4. Проверить, что в бургере есть кнопка Logout\n        \"\"\"\n        app.open_main_page()\n        app.login.auth(email, password)\n    \n>       assert (app.main_page.logout_button_text() ==\n                ButtonTexts.LOGOUT_BUTTON_TEXT, \"Кнопка Logout не найдена\")\n\ntests\\test_auth.py:32: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <pages.main_page.MainPage object at 0x0000012C05EA2E20>\n\n    def logout_button_text(self):\n>       self.logout_button().text()\nE       TypeError: 'str' object is not callable\n\npages\\main_page.py:42: TypeError"}, "description": "\n        1. Открыть страницу\n        2. Ввести валидные значения в поля логина и пароля\n        3. Нажать кнопку LOGIN\n        4. Проверить, что в бургере есть кнопка Logout\n        ", "attachments": [{"name": "screenshot", "source": "38862022-0a37-4360-a58b-cefb2774b6d9-attachment.png", "type": "image/png"}, {"name": "log", "source": "8bb69b22-eabc-4356-a0aa-a9d8fbe9a627-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "3dcaac0d-3d53-4e6c-8b2c-5b366383da10-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "email", "value": "'performance_glitch_user'"}, {"name": "password", "value": "'secret_sauce'"}], "start": 1615012532381, "stop": 1615012537859, "uuid": "23ebc358-df57-472a-9727-076e28be63a1", "historyId": "8df94a67bfcca65d5cf57a3d35530f9b", "testCaseId": "52b3ff36bc4dce34c6b761d4e4c54109", "fullName": "tests.test_auth.TestAuth#test_valid_auth", "labels": [{"name": "severity", "value": "blocker"}, {"name": "story", "value": "Авторизация"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_auth"}, {"name": "subSuite", "value": "TestAuth"}, {"name": "host", "value": "LAPTOP-Q6599UFH"}, {"name": "thread", "value": "9404-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_auth"}]}
